indica/operators/mean_charge.py

```diff
@@ -73,22 +73,17 @@ def __call__(self, FracAbundObj: DataArray, element: str):  # type: ignore
         )
         input_check("element", element, str)
 
-        element_atomic_number_tmp = [
-            k for k, v in ELEMENTS_BY_ATOMIC_NUMBER.items() if v == element
-        ]
         try:
-            assert len(element_atomic_number_tmp) == 1
+            assert element in ELEMENTS.keys()
         except AssertionError:
             raise ValueError(
                 f"Please input a single valid element from list:\
-                {list(ELEMENTS_BY_ATOMIC_NUMBER.values())}"
+                {list(ELEMENTS.keys())}"
             )
 
-        element_atomic_number = element_atomic_number_tmp[0]
+        element_atomic_number = ELEMENTS[element][0]
 
-        ionisation_charges = np.linspace(
-            0, element_atomic_number, element_atomic_number + 1, dtype=int
-        )
+        ionisation_charges = np.arange(0, element_atomic_number + 1)  # type: ignore
```

Comment:
Hmm, that's annoying about mypy throwing an error. If we wanted to un-ignore it we could also type hint in place since we know what `np.arange` has to return e.g. `ionisation_charges: np.ndarray = np.arange(...)`? No problem leaving it as ignored either if that doesn't actually gain anything for us though